dag.js

```diff
@@ -3,11 +3,12 @@ import ReactDOM from 'react-dom';
 import {configureStore} from './dag-store';
 import {getSettings} from './dag-settings';
 import uuid from 'node-uuid';
+import jsPlumb from 'jsPlumb';
```

Comment:
Yeah sorry about this. I have an [issue opened in jsplumb](https://github.com/sporritt/jsPlumb/issues/543). This will be a problem when you run unit tests. Need to figure out a way to make this uniform.